hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStreamStatistics.java

```diff
@@ -65,6 +64,12 @@
    */
   void writeCurrentBuffer();
 
+  /**
+   * Get the IOStatisticsStore instance from AbfsOutputStreamStatistics.
+   * @return instance of IOStatisticsStore which extends IOStatistics.
+   */
+  IOStatistics getIOStatistics();
```

Comment:
Is this class an instance of IOStatisticsSource? It should be.

Reply:
I had implemented IOStatisticsSource in this interface's implementation class. But, makes sense to extend here since, I am implementing this in the implementation class.